adr/2022-11-stable-spec.md

```diff
@@ -0,0 +1,180 @@
+current release and they will automatically have support for past releases.
+
+### Option 1 - TC-39 Inspired
+The two things that make this option stand out are the stability model and the
+mutability of the spec document.
+
+Having a mutable spec allows us to make clarifications and bug fixes immediately
+rather than having to wait months or years for the next release to go out. It
+also allows us to iterate faster on unstable features which would allow us to
+get them to a stable state much sooner. For example, we have changes to dynamic
+references that have been agreed upon and ready to go for over a year, but users
+can't benefit from the change until we can get the next full release published.
+With this model, the change could have been made available for over a year now
+and we would have a years worth of feedback on it's use. Having a mutable spec
+also allows us to introduce new features without having to wait for a release.
+For example, the `propertyDependencies` keyword has also been waiting for months
+for a release. Users could have been benefiting from it for months and providing
+feedback.
+
+The downside of a mutable spec is that it can be more difficult for implementers
+and users to track when changes happen. We will need to be better at
+communicating changes in blog posts or equivalent.
+
+The stability model allows us to ensure we don't make incompatible changes to
+stable features, but it also allows us to introduce new features and get real
+world feedback without committing to full compatibility requirements. This makes
+it much more likely that we don't get stuck with something that doesn't work out
+or could be done better.
+
+The stability model also makes it clear to users which features are stable and
+how likely an unstable feature is to change in the future. Whether they prefer
+to stick with stable features or want to use a new keyword, users have the
+information they need to make that decision.
+
+The downside of the stability model is that it presents a very high barrier for
+a feature to make it into a stable status. It would typically take two years for
+a feature to reach stability which could be a long time to wait for users who
+need to stick to the stable feature set but could benefit greatly from a new
+feature.
+
+### Option 2 - IETF Inspired
+The benefit of this approach is that it's compatible with the IETF process
+without imposing some of the constraints and perception issues that we had with
+IETF. We can pursue an RFC in the future if we choose to without significant
+changes or spec restructuring.
+
+With this proposal, releases are done as a new document that replaces the
+previous documents. Compared to the constantly evolving spec in Option 1, change
+from non-functional clarifications and bug fixes to adding and evolving new
+features takes much longer if you have to wait for the next release to make a
+change. This lengthens the feedback loop slowing spec development progress.
```

Comment:
This description of Option 2 is what I thought it was.  The paragraphs above doesn't describe a "document replacement" process.  Instead they merely focus on extensions.

Reply:
You're right, Option 2's "Considered Options" section is missing a lot of important details. I'll expand that section. However, Option 2 will still a focus on extensions because it proposes that extensions are the primary way for the spec to evolve.

Reply:
> However, Option 2 will still a focus on extensions because it proposes that extensions are the primary way for the spec to evolve.

I think I see where you're going with this, but perhaps being a bit more explicit on how extensions are used to evolve the spec may shed light on the difference in process between the two options.